app/forms/steps/capital/savings_form.rb

```diff
@@ -0,0 +1,56 @@
+module Steps
+  module Capital
+    class SavingsForm < Steps::BaseFormObject
+      delegate :saving_type, to: :record
+
+      attribute :provider_name, :string
+      attribute :account_balance, :pence
+      attribute :sort_code, :string
+      attribute :account_number, :string
+      attribute :is_overdrawn, :value_object, source: YesNoAnswer
+      attribute :are_wages_paid_into_account, :value_object, source: YesNoAnswer
+      attribute :account_holder, :value_object, source: OwnershipType
+
+      validates_presence_of(
+        :provider_name,
+        :sort_code,
+        :account_number,
+        :account_balance
+      )
+
+      validates_inclusion_of(
+        :is_overdrawn,
```

Comment:
Surely we can calculate this based on if they put a minus number in account_balance 🙄